src/meta/user/authorizer.ts

```diff
@@ -113,8 +113,53 @@ const canEdit = (props: {
   return false
 }
 
+/**
+ * CanEdit
+ * Viewer or non loggedin user: never
+ * Administrator: always
+ * NationalCorrespondant and AlternateNationalCorrespondant:
+ * if status is editing then true
+ * Reviewer:
+ * if status in status ('review','editing') then true
+ * @param props
+ * @param props.assessment
+ * @param props.countryIso
+ * @param props.cycle
+ * @param props.table
+ * @param props.status
+ * @param props.user
+ * @returns boolean
+ */
+const canEditAssessmentStatus = (props: {
+  countryIso: CountryIso
```

Comment:
Herr countryiso can be removed and taken from country

Reply:
done